packages/adapter-vercel/index.js

```diff
@@ -572,15 +593,26 @@ function validate_vercel_json(builder, vercel_config) {
 		unmatched_paths.push(path);
 	}
 
-	builder.log.warn(
-		`\nvercel.json defines cron tasks that use paths that do not correspond to an API route with a GET handler (ignore this if the request is handled in your \`handle\` hook):`
-	);
+	if (unmatched_paths.length) {
+		builder.log.warn(
+			`\nWarning: vercel.json defines cron tasks that use paths that do not correspond to an API route with a GET handler (ignore this if the request is handled in your \`handle\` hook):`
+		);
+
+		for (const path of unmatched_paths) {
+			console.log(`    - ${path}`);
+		}
 
-	for (const path of unmatched_paths) {
-		console.log(`    - ${path}`);
+		console.log('');
 	}
+}
 
-	console.log('');
+/** @param {import('@sveltejs/kit').RouteDefinition} route */
+function is_prerendered(route) {
+	return (
+		route.prerender === true ||
+		(route.prerender === 'auto' &&
+			!route.segments.some((segment) => segment.dynamic || segment.rest))
```

Comment:
`rest` implies `dynamic`, so we can get rid of `|| segment.rest`. i think i'd reverse how the condition is expressed — 'every segment is non-dynamic' is easier to grok than `it's not the case that some segments are dynamic'

```suggestion
			route.segments.every((segment) => !segment.dynamic))
```